src/Components/Comments.jsx

```diff
@@ -0,0 +1,35 @@
+import { useEffect, useState } from "react";
+import { getComments } from "../../api";
+
+export default function Comments({ article_id }) {
+  const [commentList, setCommentList] = useState([]);
+  useEffect(() => {
```

Comment:
Im not seeing this page on the article page hmm

Reply:
Oh no what have i done 😅 could it be because of my styling. Because it shows up on my screen